features/conviction_financial_discharge.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Conviction
+  Background:
+    When I am completing a basic under 18 "Discharge" conviction
+    Then I should see "What discharge were you given?"
+
+  @happy_path
+  Scenario: Conviction Discharge - Absolute discharge
+    And I choose "Absolute discharge"
+    Then I should see "When did you get convicted?"
+    When I enter a valid date
+
+  @happy_path
+  Scenario: Conviction Discharge - Conditional discharge
+    And I choose "Conditional discharge"
+    Then I should see "When did you get convicted?"
+
+    When I enter a valid date
+
+    Then I should see "Was the length of conviction given in weeks, months or years?"
+    And I choose "Weeks"
+
+    Then I should see "What was the length of the order?"
```

Comment:
Here I would introduce an extra step to actually ensure we are picking the correct hint text based on the selection from before, if it was months, weeks or years.

It can be done as part of this same scenario or create a new scenario only for this.